cclib/parser/gaussianparser.py

```diff
@@ -162,6 +162,12 @@ def after_parsing(self):
                 rotation = utils.get_rotation(numpy.array(inputcoord), numpy.array(atomcoord))
                 grads_std.append(rotation.apply(grad))
             self.set_attribute('grads', numpy.array(grads_std))
+        
+        if hasattr(self, "ccenergy"):
+            if not hasattr(self, "ccenergies"):
+                self.ccenergies = []
+            self.ccenergies.append(utils.convertor(self.ccenergy, "hartree", "eV"))
```

Comment:
These three lines can be replaced with
```suggestion
            self.append_attribute("ccenergies", utils.convertor(self.ccenergy, "hartree", "eV"))
```